api/config/server.js

```diff
@@ -1,4 +1,7 @@
 module.exports = ({ env }) => ({
   host: env('HOST', '0.0.0.0'),
   port: env.int('PORT', 1337),
+  app: {
```

Comment:
Not a critique, I'm just not getting what this is doing for some reason 😅

Reply:
This is needed for the [migration of 4.0.0 to 4.0.6](https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.0.x-to4.0.6.html) apparently